packages/terra-menu/src/_MenuContent.jsx

```diff
@@ -292,20 +294,22 @@ class MenuContent extends React.Component {
 
     if (this.props.index > 0) {
       header = (
-        <div
-          role="button"
-          onClick={this.props.onRequestBack}
-          onKeyDown={this.onKeyDownBackButton}
-          tabIndex="0"
-          aria-label={backBtnText}
-        >
+        <>
           <div className={cx('header-container')}>
-            <div className={cx('header-button')}>
+            <div
+              className={cx('header-button')}
+              role="button"
+              onClick={this.props.onRequestBack}
+              onKeyDown={this.onKeyDownBackButton}
+              tabIndex="0"
+              aria-label={backBtnText}
+              aria-describedby={menuHeaderId}
+            >
               {backIcon}
             </div>
-            <h1 className={cx('header-title')}>{this.props.title}</h1>
+            <h1 id={menuHeaderId} className={cx('header-title')}>{this.props.title}</h1>
```

Comment:
Can you check with rahul once if h1 for the menu header is valid or not?

Reply:
Changed the header from h1 to h2.